### ISSUE TITLE
Exchange PhantomJs

### ISSUE THREAD

**Issue by Bachstelze** [jquery/jquery-mobile#8651]
PhantomJs is no longer supported... Are there plans to exchange it?
ref #8441

**Comment by ecadius** [comment]
No, i don't think so and it seems that PhantomJS is breaking the current master builds #8648. Does anybody know a good lib to which we can migrate easy from PhantomJS?

**Comment by Bachstelze** [comment]
What do you think about [karma](https://github.com/karma-runner/karma/) based on a headless [chrome](https://developers.google.com/web/updates/2017/06/headless-karma-mocha-chai)?
We will have a closer look at our tests, but perhaps we could keep them: https://blog.johnnyreilly.com/2017/08/karma-from-phantomjs-to-headless-chrome.html
